app/controllers/rule_languages_controller.rb

```diff
@@ -0,0 +1,35 @@
+class RuleLanguagesController < ApplicationController
+  before_action :authorize!
+
```

Comment:
you will need to add the convert dashes to hyphens `before_action` here.

The following request doesn't update the `negative-rule` attribute.
```
curl --location --request PUT 'https://mobile-content-api-stage.cru.org/tool-groups/1/rules-language/1' \
--header 'Content-Type: application/vnd.api+json' \
--header 'Accept: application/vnd.api+json' \
--data '{
 "data":   {
   "type": "tool-group-rule-language",
   "attributes": {
       "negative-rule": false
   }
 }
}'
```

Reply:
Solved in commits yesterday